crates/virtio-bindings/CONTRIBUTING.md

```diff
@@ -6,7 +6,7 @@
 The bindings are currently generated using
 [bindgen](https://crates.io/crates/bindgen) version 0.61.0:
```

Comment:
nit: we should probably update this as well.

Reply:
I've changed it to link to the bindgen website, since linking to either individual crate felt a bit overly specific to me.

Reply:
👍 thank you